modules/statistics/CallStats.js

```diff
@@ -451,17 +451,17 @@ export default class CallStats {
      *
      * @param {string} conferenceID the conference ID for which the feedback
      * will be reported.
-     * @param overallFeedback an integer between 1 and 5 indicating the
+     * @param overall an integer between 1 and 5 indicating the
      * user feedback
-     * @param detailedFeedback detailed feedback from the user. Not yet used
+     * @param comment detailed feedback from the user.
      */
-    static sendFeedback(conferenceID, overallFeedback, detailedFeedback) {
+    static sendFeedback(conferenceID, overall, comment) {
```

Comment:
if we're gonna rename it, how about something like `score` instead of `overall` ?

Reply:
I was going with "rating" and "comment", but I saw that the callstats API calls it "overall", so I kept it this way.
https://www.callstats.io/api/#callstats-senduserfeedback